tests/virtualization/vagrant/add_box.pm

```diff
@@ -0,0 +1,44 @@
+# Copyright (C) 2019 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+# Summary: Test for vagrant and packaged addons
+# Maintainer: dancermak <dcermak@suse.com>
+
+use strict;
+use warnings;
+use base "basetest";
+use testapi;
+use utils;
+
+sub run() {
+    select_console('root-console');
+
+    zypper_call('in vagrant virtualbox');
+    assert_script_run('systemctl start vboxdrv');
+    assert_script_run('systemctl start vboxautostart');
+    assert_script_run('usermod -a -G vboxusers bernhard');
+
+    select_console('user-console');
+    assert_script_run('echo "test" > testfile');
+
+    # TODO: replace this with a openSUSE box once we have officially released boxes
```

Comment:
Better leave that comment out and keep in your local git repository, either in `git stash` or a temporary "WIP"-commit, or just have a ticket for that. It is not really something we can work on now.

Reply:
I'll drop it then, should I create a github issue for that?

Reply:
Comment has been dropped.